contracts/JBProjects.sol

```diff
@@ -140,18 +133,17 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
   */
   function tokenURI(uint256 _projectId) public view override returns (string memory) {
     if (tokenUriResolver == IJBTokenUriResolver(address(0))) return '';
-
     return tokenUriResolver.getUri(_projectId);
   }
 
   /**
     @notice 
-    Set the address of the IJBTokenUriResolver used to retrieve the tokenURI of projects
+    Set the address of the IJBTokenUriResolver used to retrieve the tokenURI of projects.
 
     @param _newResolver The address of the new resolver.
   */
   function setTokenUriResolver(IJBTokenUriResolver _newResolver) external override onlyOwner {
     tokenUriResolver = _newResolver;
-    emit SetTokenUriResolver(_newResolver);
+    emit SetTokenUriResolver(_newResolver, msg.sender);
```

Comment:
Same with event and this fn name.

Reply:
this is the standard. we cant change the overriden tokenURI one though. The rest of the JB ecosystem prefers the camel case version.

should we keep consistent in the contract, or in the JB ecosystem?

Reply:
Ah yes you're right, sorry forgot about that. camelCaseSoundsGoodToMe